scripts/run_msf_addons.sh

```diff
@@ -50,4 +50,33 @@ fi
 
 # this step requires internet. It downloads node modules. Should be run on the host
 echo "$INFO Pre-warming worker cache with adaptors..."
-docker exec -it $PROJECT_NAME-worker-1 sh -c "npm install -g @openfn/cli && openfn repo install  -a common@latest -a collections@latest -a http@6.5.1 -a openmrs@4.1.3 -a dhis2@5.0.1"
```

Comment:
I would advise that we still prewarm the worker cache

Reply:
@midigofrank, the latest of @openfn/cli ie 1.10.3 didn't work for me.  I was getting an error, `extraneous parameters: collections`.  See the error log attached below. 

As a fix for that i downgraded to 1.10.2 which solve that for me.